plugins/cost-insights/src/components/AlertInsights/AlertInsightsHeader.tsx

```diff
@@ -36,9 +36,9 @@ export const AlertInsightsHeader = ({
       <ScrollAnchor id={DefaultNavigation.AlertInsightsHeader} />
       <Typography variant="h4" align="center">
         {title}{' '}
-        <span role="img" aria-label="direct-hit">
+        <Typography component="span" role="img" aria-label="direct-hit">
```

Comment:
Just curious, do you recommend any other component for this case, where we mean role image?
Would you recommend adding an aria-hidden="true" or using another component instead of the Typography?

Reply:
Most examples in MDN docs use `div`, not my area of expertise though...